docs/specification/1.0.md

```diff
@@ -383,35 +383,64 @@ To reduce the implementation burden on EHRs that support CDS services, CDS Hooks
 
 ### Trusting CDS Services
 
-As the EHR initiates every interaction with the CDS Service, it is responsible for establishing trust with the CDS Services it intends to call. This trust is established via a TLS connection to the CDS Service. Thus, all CDS Service endpoints must be deployed to a TLS protected URL (https). This includes both the Discovery and individual CDS Service endpoints.
+Prior to enabling EHRs to request decision support from any CDS Service, the EHR vendor and/or provider organization is expected to perform due diligence on the CDS Service provider.  Some EHR vendors and provider organizations may maintain a “white list” of CDS Service providers they have deemed trustworthy.  
 
-EHRs should use accepted best practices for verifying the authenticity and trust of these TLS connections. For instance, [rfc5280](https://tools.ietf.org/html/rfc5280) and [rfc6125](https://tools.ietf.org/html/rfc6125). Additionally, it is assumed that EHRs configure the CDS Services they connect to via some offline process according to the business rules and practices of both the EHR and CDS Service organizations.
+Once a CDS Service provider is selected, the EHR vendor/provider negotiates the terms under which service will be provided.  This negotiation includes agreement on patient data elements that will be prefetched and provided to the CDS Service, data elements that will be made available through an access token passed by the EHR, and steps the CDS Service must take to protect patient data and access tokens.  To enable the EHR authorization server to authorize CDS Service access to FHIR resources, the CDS Service is registered as a client to the EHR authorization server.  These business agreements are documented in the service agreement.  Because the CDS Service will receive patient data and will provide clinical decision support, the Service provider also will need to sign a Business Associate Agreement (BAA) that commits it to compliance with the Privacy and Security Rules of the Health Insurance Portability and Accountability Act (HIPAA).   
+
+Every interaction between an EHR and a CDS Service is initiated by the EHR sending a service request to a CDS Service endpoint protected using the [Transport Layer Security protocol](TLS, RFC5246)[ https://tools.ietf.org/html/rfc5246].   Through the TLS protocol the identity of the CDS Service is authenticated, and an encrypted transmission channel is established between the EHR and the CDS Service. Both the Discovery endpoint and individual CDS Service endpoints are TLS secured.
 
```

Comment:
This paragraph implies that mutual TLS is required, but subsequent paragraphs make clear that mutual TLS is optional as it's a function of the security environment. Suggest dropping the last sentence.

Reply:
@brynrhodes - I don't see this paragraph talking about _mutual_ TLS. It reads to me that we are just talking about TLS here.

Can you elaborate on the part that makes you think it is regarding _mutual_ TLS?

Reply:
Oh, you're right, disregard, I read the "both" as somehow applying to the EHR and CDS Service, but I see they are both endpoints exposed by the CDS Service.

Reply:
@kpshek @brynrhodes  Right, server-point TLS is all that is needed to conform to the assertion in the last sentence.